src/openassetio-core/include/openassetio/EntityReference.hpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 The Foundry Visionmongers Ltd
+#pragma once
+
+#include <utility>
+
+#include <openassetio/export.h>
+#include <openassetio/typedefs.hpp>
+
+namespace openassetio {
+inline namespace OPENASSETIO_CORE_ABI_VERSION {
+/**
+ * EntityReference forms a strongly typed wrapper around arbitrary
+ * strings to ensure they have been validated by the target @ref manager
+ * before being used as an @ref entity_reference in the various entity
+ * related API calls.
+ *
+ * It can be assumed that if true is returned from @ref
+ * openassetio.hostApi.Manager.Manager.isEntityReference
+ * "isEntityReference" is True for any given string, then an
+ * EntityReference can be constructed from that string.
```

Comment:
[fix] "if true is returned from .... is True for"

Reply:
Tweaked in 68c3bf68

Reply:
Doh! Thanks 🙄